app/app.go

```diff
@@ -531,6 +534,10 @@ func NewEvmos(
 	app.ScopedIBCKeeper = scopedIBCKeeper
 	app.ScopedTransferKeeper = scopedTransferKeeper
 
+	// Finally start the tpsCounter.
+	app.tpsCounter = newTPSCounter(logger)
+	go app.tpsCounter.start(context.Background())
```

Comment:
q: does this always run by default?

Reply:
It does run by default indeed. Would you like to gate it instead?

Reply:
yes, ideally it should be optional for nodes